client/app/configuration/deployment-maps/DeploymentMapTargetController.js

```diff
@@ -282,6 +287,15 @@ angular.module('EnvironmentManager.configuration').controller('DeploymentMapTarg
       }
     }
 
+    function getDefaultCluster() {
+      var owningCluster = teamstorageservice.get('');
+      return owningCluster === 'Any' ? '' : owningCluster;
+    }
+
+    function getDefaultClusterEmail(defaultCluster) {
+      return defaultCluster === '' ? '' : getClusterByName(defaultCluster).Value.GroupEmailAddress;
+    }
```

Comment:
getClusterByName will return an empty list if there is no matching value. Can we handle there being no `Value` here. Or better yet, wrap it up in a `getClusterEmailByName()`

Reply:
👍